### DISCUSSION TITLE
Didnt work EXPAND option for single record in Rest API

### DISCUSSION THREAD

**Discussion by dgotskin** [pocketbase/pocketbase#4301] [Q&A] [answered]
Hi,
Please help to understand...🙏

When I make a request on the admin panel for a list with the expand option, I get the desired result. The related objects specified by the field are returned as expanded objects. However, when I make a request for a single record with the same option, it doesn't work... I've tried all syntaxes and checked the permissions in the settings...

`/api/collections/photos/records?sort=-created&expand=tags&page=1&perPage=1000` is work correctly
`/api/collections/photos/records/4qiewyp0tik8bmu?expand=tags` doesnt work

In first case I do request with admin rights at Admin Panel
In second case requested via client side in app.

(tags is relation field)

API Rules for Tags collection is "allow for all" (no restrictions)

[PocketBase v0.21.1]

**Comment by dgotskin** [comment]
![image](https://github.com/pocketbase/pocketbase/assets/6461264/4521f59e-707d-4ef7-a81d-9b87c792ccb4)

**Reply by ganigeorgiev** [reply, marked as answer]
Double check the View API rules of the expanded collections.

If you are still not able to resolve it, please paste your collections configuration as JSON (_Admin UI > Settings > Export collections_).

**Reply by dgotskin** [reply]
![image](https://github.com/pocketbase/pocketbase/assets/6461264/bf3dbd9f-e940-43d4-b879-a132df69dc8f)

I'm sorry, I'm such a fool! When I took this screenshot, I suddenly realized that the inscription "For administrators only" represents the current state of the rule. Before, I thought it would be activated if I clicked on that field. In short, I misunderstood the interface, and now that I switched to another mode, everything is working fine. Thank you, I'm very inattentive. You can close this ticket.
